src/plugin/regions.js

```diff
@@ -319,8 +319,9 @@ class Region {
                     );
 
                     // Update time
-                    const time =
-                        this.wavesurfer.drawer.handleEvent(e) * duration;
+                    const time = this.wavesurfer.regions.util.getRegionSnapToGridValue(
```

Comment:
add comment what value `time` is expected to be

Reply:
time has not been introduced with this PR. I tried to explain it better.